.github/workflows/spell_check_pr.yml

```diff
@@ -0,0 +1,23 @@
+name: Check spelling
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  spellcheck:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Retrieve spell check dictionary
+        run: |
+          curl --silent --show-error \
+            --output .github/workflows/.cspell.json \
+            https://raw.githubusercontent.com/tier4/autoware-spell-check-dict/main/cspell/.cspell.json
+
+      - uses: streetsidesoftware/cspell-action@v1.1.1
```

Comment:
[nits] Can be updated. Installing dependabot is useful.

Reply:
Add dependabot and update cspell-action version. https://github.com/tier4/ars408_driver/pull/1/commits/fdf462278ddf0aab92b573ee7bf04437c1f9eadf
https://github.com/tier4/ars408_driver/pull/1/commits/bbc1279cecf2ae48a9cc55e4892ab5019d337c9d